### ISSUE TITLE
[Feature Proposal] Custom Instructions

### ISSUE THREAD

**Issue by secondtruth** [huggingface/chat-ui#748]
I propose the introduction of a "Custom Instructions" feature like in ChatGPT. This feature would allow users to tailor model and assistant responses to their individual preferences for style, formality, and output format. Also, by storing custom instructions in user settings, time spent for adjusting the chatbot every time for each model/assistant is saved.

**Comment by nsarrazin** [comment]
so if I understand correctly this is different from system prompts because it's set globally for every models ?

**Comment by secondtruth** [comment]
Yes, it's basically a globally set extension that will be added to the system prompt of every model.

This is the official blog post of the feature: https://openai.com/blog/custom-instructions-for-chatgpt
